locales/en-US/messages.properties

```diff
@@ -31,7 +31,7 @@ discourse_forum=Discourse forum
 teaching_activities=Teaching Activities
 web_literacy=Web Literacy
 web_lit_skills=Web Literacy Skills
-web_lit_intro=A framework for entry-level web literacy &amp; 21st Century skills. Explore the map by selecting what you want to learn more about, to see definitions and activities.
+web_lit_intro=A framework for entry-level web literacy & 21st Century skills. Explore the map by selecting what you want to learn more about, to see definitions and activities.
```

Comment:
I'm not sure I fully understand this change, shouldn't react-intl accept html entities in strings?